ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/systemcall/SystemCallProcessor.java

```diff
@@ -37,6 +37,13 @@
 public class SystemCallProcessor {
   private static final Logger LOG = LoggerFactory.getLogger(SystemCallProcessor.class);
 
+  /**
+   * The gas limit as defined in <a
+   * href="https://eips.ethereum.org/EIPS/eip-2935#block-processing">EIP-2935</a> This value is
+   * independent of the gas limit of the block
+   */
+  private static final long SYSTEM_CALL_GAS_LIMIT = 30_000_000L;
```

Comment:
if anything triggers an issue in pectra, I bet it would be some client using the block limit for system call gas limit